### ISSUE TITLE
Errors with release 3.16

### ISSUE THREAD

**Issue by fortune9** [Bioconductor/bioconductor_docker#61]
The docker image has the following issues when I run the commands within the container:

- Command: _R_
  * error message:
    OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
    OpenBLAS blas_thread_init: RLIMIT_NPROC -1 current, -1 max

- Command: _apt-get update_
  * error message:
     Reading package lists... Done
     E: Problem executing scripts APT::Update::Post-Invoke 'rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true'
     E: Sub-process returned an error code

Environment:

- BIOCONDUCTOR_DOCKER_VERSION : 3.16.31
- My host system: 18.04.1-Ubuntu

Please advise how to address the above issues.

**Comment by vjcitn** [comment]
Please provide the command you used to invoke docker on the container.  You say the command "R" fails in a container started with docker run ... bash?  Please give the version of your docker client also.  Thanks.

**Comment by fortune9** [comment]
Thank you for the quick reply, @vjcitn .

I initiated the docker container using the following command:

> docker run -it --rm bioconductor/bioconductor_docker:RELEASE_3_16 bash

And then run the commands in my first post.

**Update**

Actually, after I posted the issue here, I did some google search and found a fix.

The issue seemed due to my old version of docker, which was 20.10.8. After I updated the docker engine to 20.10.21
[instructions here](https://docs.docker.com/engine/install/ubuntu/), the issue was fixed.

Thank you for your help all the same!
